pkg/sql/catalog/lease/kv_writer.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package lease
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/keys"
+	"github.com/cockroachdb/cockroach/pkg/kv"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/bootstrap"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/systemschema"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/tabledesc"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/errors"
+)
+
+// kvWriter implements writer using the raw KV API.
+type kvWriter struct {
+	db *kv.DB
+	w  bootstrap.KVWriter
+}
+
+func newKVWriter(codec keys.SQLCodec, db *kv.DB, id descpb.ID) *kvWriter {
+	return &kvWriter{
+		db: db,
+		w:  bootstrap.MakeKVWriter(codec, leaseTableWithID(id)),
+	}
+}
+
+func leaseTableWithID(id descpb.ID) catalog.TableDescriptor {
+	mut := systemschema.LeaseTable.NewBuilder().
+		BuildExistingMutable().(*tabledesc.Mutable)
+	mut.ID = id
+	return mut.ImmutableCopy().(catalog.TableDescriptor)
+}
```

Comment:
nit: add a comment that this function exists to facilitate testing, perhaps with a fast exit:
```
func leaseTableWithID(id descpb.ID) catalog.TableDescriptor {
  if id == systemschema.LeaseTable.GetID() {
    return  systemschema.LeaseTable
  }
  // blah blah testing
```

Reply:
Done.